models/schemas/OperationOutcome-2XX.yaml

```diff
@@ -0,0 +1,32 @@
+type: object
+description: |
+  Outcome of an operation that does not result in a resource or bundle being returned (e.g. error, async/batch submission).
+properties:
+  resourceType:
+    type: string
+    description: FHIR Resource Type.
+    default: "OperationOutcome"
+    readOnly: true
+  issue:
+    type: array
+    description: List of issues that have occurred.
+    minItems: 1
```

Comment:
is there always an issue?

Reply:
Yes, in the fhir spec, it is a required element.